src/test/java/com/mongodb/InterruptibilityItTest.java

```diff
@@ -105,8 +105,6 @@ void beforeEach() {
 
     @AfterEach
     void afterEach() {
-        // make sure the current thread is not interrupted because tests mess with this status
-        com.mongodb.assertions.Assertions.assertFalse(Thread.interrupted());
```

Comment:
Removed this assertion because if the test framework were to interrupt tests, the assertion would have incorrectly failed.